### ISSUE TITLE
How can multi-stream boost the acc so much? 

### ISSUE THREAD

**Issue by YujieOuO** [Levigty/AimCLR#9]
Any extra parameters or losses are used during 3-s pretraining and downstream?

**Comment by Levigty** [comment]
For the multi-stream, we just fuse the result in this paper, and no additional loss is used. In the fully supervised task, the results of three streams are also better than that of one stream, we think it is because the information between the three streams has complementarity. In our recent work, we also tried the joint optimization of three streams, which can achieve better performance. But in this article we only simply fuse the results.

**Comment by YujieOuO** [comment]
Thanks for your reply, 
so, you just train the joint , bone and motion data respectively and use the ensemble_cs.py to fuse the result?
Could you please tell me the input of the ensemble_cs.py?
Is it 256-dim feature or 60-dim score or 60-dim probability?

**Comment by YujieOuO** [comment]
Actually, i am a little curious about the word [we train a linear classiﬁer (a fully-connected layer followed
by a softmax layer)] in your experiment. We know the softmax layer may be not necessary because the 
cross entropy loss can handle the 60-dim score but not probability.

**Comment by YujieOuO** [comment]
i get it , the result is correct.
